.rubocop.yml

```diff
@@ -20,6 +20,7 @@ Style/Documentation:
   Enabled: false
 
 Metrics/LineLength:
+  Max: 200
```

Comment:
This will make RuboCop force lots of conditionals to use postfix syntax, so better keep this shorter, say at 92 or 100.

Reply:
Why do we need this at all? The syntax of the Ruby language doesn’t generally lead to long lines anyway, so I tend to think a style cop for it would be excessive (and difficult for those few but nonzero cases where the long line is the preferable option).

Reply:
See SHA - Standardisation across repos https://github.com/cucumber/cucumber-rails/pull/434/commits/40fc5b25479c00ccdd55449663e90b529192dafd

Reply:
Well, I think the reasoning in `cucumber-ruby` is wrong. Line length in this github issue view is a lot less than 200.

I think @marnen is asking why this cop is not simply disabled? I'm afraid there are still people who needlessly put a lot of stuff on a single line. This is bad for readability so some check is called for.

Reply:
@mvz Personally I’d rather catch that in code review. There can be 30-character lines that are too long and (occasionally) 120-character lines that are not too long. I don’t think a hard character limit does any good.

Reply:
There could always be exceptions, but for the majority of cases a hard character limit is useful for automating the boring part of code review.

I would never flag a 30-character line as just 'too long', but rather something else would be wrong with it, like 'too complex', or 'multiple statements on one line'.

Reply:
@mvz If there can always be exceptions, then is this suitable for cop automation? I prefer to automate only those rules that we absolutely never want to violate, because I don’t like merging red code into the project.

Reply:
Exceptions will have to be marked as such to avoid the build failing.

Reply:
Note that I think exceptions for line length are highly unlikely, and I think the reasons for making such an exception would have to be really strong indeed.

Reply:
@mvz As do I. But if the cop is set to 200 characters, we might as well not have it at all: no one would ever submit a pull request with a line that long in the first place, and if they did, it would be immediately visually obvious.

Reply:
@marnen I'm not arguing for 200 chars.

Reply:
@luke-hill if you prefer to keep it at 200 for consistency, I'm OK with that.

Reply:
If the cucumber-ruby or other ruby-ish ones change it. I'll change it here. I think that's a debate for another day. I'm for standardisation

Reply:
@luke-hill agreed.